app_server/app/controllers/UserController.js

```diff
@@ -134,6 +159,14 @@ var loginPayloadValidator = {
   }
 };
 
+var updateUserValidator = {
+  payload: {
+    alias: Joi.string().optional().default(null),
+    description: Joi.string().optional().default(null),
```

Comment:
`alias` shouldn't be optional or null. Otherwise we may not have anything to display about the user.